### ISSUE TITLE
Error 712: Unable to access keybase team folders.

### ISSUE THREAD

**Issue by trev-dev** [keybase/client#13950]
I've been trying to debug keybase on my own and I'm throwing in the towel and asking for help.

I get several errors while trying to access my keybase team folders. I cannot access them in the finder. I cannot ls their contents in the command prompt. I _can access my private and public files_ without any issues.

Using [keybase-bin](https://aur.archlinux.org/packages/keybase-bin/) on Arch Linux. I tried installing the packages from the official repositories and following the directions on [the wiki](https://wiki.archlinux.org/index.php/Keybase) but kbfs wouldn't even mount with those packages.

Currently this is where I'm at on this machine:

![screenshot_20180926_151144](https://user-images.githubusercontent.com/28788713/46112225-c93cd200-c19e-11e8-9e99-bba3ba93efa9.png)
![screenshot_20180926_151213](https://user-images.githubusercontent.com/28788713/46112226-c93cd200-c19e-11e8-915e-fadc7b4900f4.png)
![screenshot_20180926_151312](https://user-images.githubusercontent.com/28788713/46112227-c93cd200-c19e-11e8-84ce-1c069282c0df.png)

KBFS is working fine on my mobile device and on my Manjaro desktop using the same keybase-bin package.

my log id: 783ab85dd519e6109c83db1c

**Comment by maxtaco** [comment]
Sorry about this, looking into it

**Comment by maxtaco** [comment]
Ok, that feature was flagged off on the server, but the clients are out, so it needs to go out too.  Can you restart your keybase to clear the cache?  Then it should work. LMK if not.

**Comment by trev-dev** [comment]
I'm about 95% sure that I did this to myself somehow, but I appreciate the apology. At one point I managed to upset keybase badly enough to have to de-authorize and re-authorize my laptop to get it to work. I'm sure somewhere along the way I messed this up, maybe?

![screenshot_20180926_160234](https://user-images.githubusercontent.com/28788713/46114064-df01c580-c1a5-11e8-9a22-464150d865ca.png)

Either way we're looking good! Thanks for the help.
